.github/workflows/docker_build.yml

```diff
@@ -1,18 +1,59 @@
-name: Publish Docker
-on: 
+# -------------------------------------------------------------------------------------------------
+# Job Name
+# -------------------------------------------------------------------------------------------------
+name: Build-Publish
+
+# -------------------------------------------------------------------------------------------------
+# When to run
+# -------------------------------------------------------------------------------------------------
+on:
   push: {}
   schedule:
     - cron:  '0 0 * * *'
+
+# -------------------------------------------------------------------------------------------------
+# What to run
+# -------------------------------------------------------------------------------------------------
 jobs:
   build-publish:
+    name:  "[ ${{ matrix.TF_VERSION }}-${{ matrix.TG_VERSION }} ]"
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: False
+      matrix:
+        TF_VERSION: [0.12,0.13,latest]
+        TG_VERSION: [0.21.7,0.22,latest]
+        exclude:
+          - { TF_VERSION: latest, TG_VERSION: 0.21.7 }
+          - { TF_VERSION: 0.13,   TG_VERSION: 0.21.7 }
```

Comment:
```suggestion
          - { TF_VERSION: 0.13,   TG_VERSION: 0.21 }
```

Reply:
We are still tied to version `0.21.7` if I remove this version, we will jump to the latest patch level of`0.21`